src/directory/directory.js

```diff
@@ -289,8 +289,8 @@ const directory = {
             filters: ['android', 'flutter', 'ios']
           },
           {
-            title: 'Social sign in with web UI',
```

Comment:
I thought this was because the way is defined by mobile platforms. I would check with iOS and Android teams about this

Reply:
Maybe inside the Android / iOS / Flutter pages we could mention the `web UI` key word if this is required.
Will confirm with the receptive teams for the same

Reply:
Checked, this shouldn't be a concern